### DISCUSSION TITLE
SB5.1 - OCI - Ansible variables clarifications

### DISCUSSION THREAD

**Discussion by ddevadat** [Sunbird-Ed/Community#397] [Installation] [answered]
We are installing rel 5.1 in OCI

Need a few clarification on the below variables

in Core/common.yaml , we need to set below variables
cloud_storage_schema_bucketname
cloud_storage_offlineinstaller_bucketname
cloud_storage_termsandcondtions_bucketname
cloud_storage_certservice_bucketname

Could you please let us know if these buckets needs to be public or private

**Comment by saiakhil46** [comment, marked as answer]
@ddevadat Please make it as public read access
